vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -176,20 +161,21 @@ kubectl apply -f namespace-resources.yaml
 helm template quickstart ./ > my-clustered-app.yaml
 ```
 
-11. Add environment variables to the `EntandoApp` in the outcome of the Helm template command generated above (`my-clustered-app.yaml`) for the Redis address and Redis password for your deployed Redis instance. The variables to create are:
+11. Add environment variables to `EntandoApp` for your deployed Redis instance per the output of the Helm template command above (`my-clustered-app.yaml`). The variables to create are:
```

Comment:
I think "per" complicates the sentence. Maybe simplify to "for". Not sure, but has the meaning changed? Do you put the environment variables in "my-clustered-app.yaml? In this edit, it sounds like you put the env var in "EntandoApp" with the information from my-clustered-app.yaml.

Reply:
good catch. the way it was worded it seemed like that's what was meant, but reading this like i'm trying to follow an instruction it's obv you're supposed to edit the yaml - how would you even put a variable in EntandoApp?? -_-